Projects/DesignSystem/Sources/Component/PaddingLabel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PaddingLabel.swift
+//  DesignSystem
+//
+//  Created by 리아 on 2022/10/24.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+
+final public class PaddingLabel: UILabel {
+    
+    private var padding = UIEdgeInsets(top: 6.0, left: 10.0, bottom: 6.0, right: 10.0)
```

Comment:
p2;

ReviewCardView에서 categoryLabel의 높이를 20으로 지정해주기 때문에 상하 패딩은 의미가 없을 것 같습니다.
그리고 Xcode로 좀 만져봤는데 좌우 패딩값은 8이 제일 나은거 같네요 변경해주시면 감사하겠습니다🙇‍♂️

```suggestion
    private var padding = UIEdgeInsets(top: 0, left: 8.0, bottom: 0, right: 8.0)
```

Reply:
[2a6a088](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/49/commits/2a6a0886a08a21d596fdeec1c5e4999b47979211) 반영 완료!